var/spack/repos/builtin/packages/qhull/package.py

```diff
@@ -47,8 +48,7 @@ class Qhull(Package):
     
     depends_on('cmake')
 
-    def install(self, spec, prefix):
-        with working_dir('spack-build', create=True):
-            cmake('..', *std_cmake_args)
-            make()
-            make("install")
+    @CMakePackage.sanity_check('build')
+    @CMakePackage.on_package_attributes(run_tests=True)
```

Comment:
Is it setup to automatically build in 'spack-build' or '.'? A lot of packages refuse to build in the root directory. A lot also refuse to build in a different directory. I'm not a big fan of CMake...